### ISSUE TITLE
doc tarball contains object code

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1412]
It was pointed out by @juippis in https://github.com/gentoo/gentoo/pull/19402 that our doc tarball contains object code and other undesirable taints. Fix up the packaging here. Probably best to reupload the 2.2.3 doc package, as well, so that we can get a package into Gentoo. See #746 .

**Comment by dankamongmen** [comment]
Sure enough:

```
libnotcurses++.so.2.2.3
libnotcurses-core.so.2.2.3
libnotcurses.so.2.2.3
```

Everything else looks fine.

**Comment by dankamongmen** [comment]
I've updated the 2.2.3 doc tarball.

**Comment by dankamongmen** [comment]
Alright, this ought be fixed for the upcoming 2.2.4 release.
